src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2698,7 +2698,9 @@ private void ReportTypeLayoutUpdateRudeEdits(List<RudeEditDiagnostic> diagnostic
 
         private static bool HasBackingField(IEventSymbol @event)
         {
+#nullable disable // https://github.com/dotnet/roslyn/issues/39288
             return @event.AddMethod.IsImplicitlyDeclared
```

Comment:
The doc on `AddMethod` says "Null only in error scenarios.". The EnC analyzer does not perform semantic analysis in documents that have syntax errors. It would be useful to clarify what "error scenarios" means - is it syntax errors in the event declaration or could it also be a semantic error? I'd think that the symbols will be non-null in presence of semantic errors (error symbols might be present in the method symbol signature, but it won't be null).

Reply:
I'm going to leave this for a follow-up change per #39288

Reply:
@gafter @333fred @agocke Could you please clarify what are the "error scenarios" mentioned in doc comments of `IEventSymbol.AddMethod`? Are these syntactic or semantic errors?

Reply:
The only time I'm aware of it happening is when the event has an accessor list that is missing an accessor.

Reply:
I see. Seems like we may have a bug here.